lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -521,7 +507,17 @@ export class PublicClientApplication {
      * @returns {string} post logout redirect URL
      */
     public getPostLogoutRedirectUri(): string {
-        return this.authModule.getPostLogoutRedirectUri();
+        if (this.config.auth.postLogoutRedirectUri) {
+            if (typeof this.config.auth.postLogoutRedirectUri === "function") {
+                return this.config.auth.postLogoutRedirectUri();
+            } else if (
+                !StringUtils.isEmpty(this.config.auth.postLogoutRedirectUri)
```

Comment:
Given the truthy check about for `postLogoutRedirectUri`, this statement will never be false, and can be an `else` instead.

Reply:
done.